### PR TITLE
Add PodSecurityPolicy for falcosidekick

### LINKED ISSUE #110: falcosidekick does not provide PodSecurityPolicy
**Describe the bug**

The helm chart has a switch to create a PodSecurityPolicy: `podSecurityPolicy.create`
But this switch only adds a ClusterRole and RoleBinding.

**How to reproduce it**

Install helm Chart with `podSecurityPolicy.create=true`

**Expected behaviour**

Creates a matching PodSecurityPolicy for falcosidekick.

**Environment**

- Falco version:
falcosidekick version: 2.14.0
falcosidekick helm chart version: 0.1.25
- Installation method:
helm

### PR DESCRIPTION
**Pull request by czunker**
Fixes #110

**What type of PR is this?**

/kind bug
/kind chart-release

**Any specific area of the project related to this PR?**

/area falcosidekick

**Special notes for your reviewer**:

**Checklist**

- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] CHANGELOG.md updated

### CODE REVIEW

**Review by leogr — CHANGES_REQUESTED** [review]
The changelog needs to be updated too.
Could you fix it, please?

Thanks in advance

/cc @Issif

**Comment by czunker** [comment]
Hi,

thanks for the review.
I added docs and rebased.

File: falcosidekick/README.md [hunk 1]
```diff
@@ -151,6 +151,7 @@ The following table lists the configurable parameters of the Falcosidekick chart
...
 | `azure.eventhub.namespace`       | Name of the space the Hub is in                                                                                                                                           |                                                                                                   |
 | `azure.eventhub.minimumpriority` | minimum priority of event for using use this output, order is `emergency\|alert\|critical\|error\|warning\|notice\|informational\|debug or ""`                                   | `debug`                                                                                           |
+| `podSecurityPolicy.create`       | create a PodSecurityolicy                                                                                                                                                 | `false`                                                                                           |
```
Comment (Issif) [inline falcosidekick/README.md:154]:
```suggestion
| `podSecurityPolicy.create`       | create a PodSecurityPolicy                                                                                                                                                 | `false`                                                                                           |
```

**Review by Issif — CHANGES_REQUESTED** [review]

File: falcosidekick/README.md (hunk 1 above)
Comment (czunker) [reply falcosidekick/README.md:154]: Good catch. Thanks.

**Review by Issif — APPROVED** [review]

**Review by Issif — APPROVED** [review]

**Review by leogr — APPROVED** [review]

**Pull request merged**